lib/msal-core/src/UserAgentApplication.ts

```diff
@@ -460,23 +501,30 @@ export class UserAgentApplication {
   acquireTokenRedirect(scopes: Array<string>, authority: string, user: User): void;
   acquireTokenRedirect(scopes: Array<string>, authority: string, user: User, extraQueryParameters: string): void;
   acquireTokenRedirect(scopes: Array<string>, authority?: string, user?: User, extraQueryParameters?: string): void {
+    // Throw error if callbacks are not set before redirect
```

Comment:
this answers my question,
Does this only matter for redirect? there is a callback that needs to be set still for loginpopup right